Formula/r/rattler-index.rb

```diff
@@ -0,0 +1,27 @@
+class RattlerIndex < Formula
+  desc "Index conda channels using rattler"
+  homepage "https://github.com/conda/rattler"
+  url "https://github.com/conda/rattler/archive/refs/tags/rattler_index-v0.21.0.tar.gz"
+  sha256 "ae98caa85a6d66ffdb80dc5176a08e4972816db4841e481cc5372792b8857114"
+  license "BSD-3-Clause"
+  head "https://github.com/conda/rattler.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "openssl@3"
+  end
+
+  def install
+    cd "crates/rattler_index" do
+      system "cargo", "install", *std_cargo_args
+    end
```

Comment:
rather than doing cd, maybe just use path to ref the cargo.toml?

```suggestion
    system "cargo", "install", *std_cargo_args(path: "crates/rattler_index")
```